src/chainparams.h

```diff
@@ -91,6 +93,7 @@ class CChainParams
     bool fMineBlocksOnDemand;
     CCheckpointData checkpointData;
     ChainTxData chainTxData;
+    bool m_fallback_fee_enabled;
```

Comment:
Nit, could be after `fMineBlocksOnDemand` above.

Reply:
Can you explain why?